doc/how-to-use-newconnector.md

```diff
@@ -0,0 +1,32 @@
+# How to use the Connector object
+
+A Connector holds information in a DSN and is ready to make a new connection at any time. Connector implements the database/sql/driver Connector interface so it can be passed to the database/sql `OpenDB` function. Settings that cannot be passed through a DSN string can be set directly on the Connector by using the `SessionInitSQL` member of the Connector.
```

Comment:
This is great. Current wording however makes it seem that SessionInitSQL is the only option available, but there are others, though less common. 

One property on the Connector is the SessionInitSQL field...